nautobot/extras/navigation.py

```diff
@@ -372,6 +372,22 @@
                             ),
                         ),
                     ),
+                    NavMenuItem(
+                        link="extras:externalintegration_list",
+                        name="External Integrations",
+                        weight=300,
+                        permissions=[
+                            "extras.view_externalintegration",
+                        ],
+                        buttons=(
+                            NavMenuAddButton(
```

Comment:
~No bulk import view for this model?~
Should probably add a button to bulk import here right?